internal/app/go-filecoin/node/node.go

```diff
@@ -405,6 +405,19 @@ func (node *Node) SetupMining(ctx context.Context) error {
 		}
 	}
 
+	if err := node.StorageMining.Start(ctx); err != nil {
+		fmt.Printf("error starting storage miner: %s\n", err)
+	}
+
+	if err := node.StorageProtocol.StorageProvider.Start(ctx); err != nil {
+		fmt.Printf("error starting storage provider: %s\n", err)
+	}
+
+	// TODO: Retrieval Market Integration
+	//if err := node.RetrievalProtocol.RetrievalProvider.Start(); err != nil {
+	//	fmt.Printf("error starting retrieval provider: %s\n", err)
+	//}
```

Comment:
@shannonwells is there still something missing here?

Reply:
No, that comment is left from much earlier TODOs and can be removed. There is no "Start," only setup.

Reply:
deleted